src/main/java/org/folio/inventory/client/OrdersClient.java

```diff
@@ -0,0 +1,74 @@
+package org.folio.inventory.client;
+
+import static java.util.Objects.isNull;
+
+import java.net.URL;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.client.WebClient;
+import lombok.SneakyThrows;
+import org.apache.http.HttpStatus;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import org.folio.inventory.common.Context;
+import org.folio.inventory.dataimport.exceptions.OrdersLoadingException;
+import org.folio.inventory.support.http.client.OkapiHttpClient;
+
+public class OrdersClient {
```

Comment:
Interface dependency should be added to ModuleDescriptor, probably in optionalRequired section

Reply:
Also, related permissions should be added to mod-source-record-manager and mod-data-import. I suggest to create related stories.